lib/sgtn-client/api/translation.rb

```diff
@@ -86,6 +86,7 @@ def self.getTranslation(component, key, locale)
       end
 
       def self.get_cs(component, locale=nil)
+        locale = locale.to_s
```

Comment:
An exception will throw when locale is nil.

Reply:
No, it won't. Test nil.to_s and it didn't raise an exception

Reply:
Yes. You are right.

if locale is empty string, then it can't be set to SgtnClient::Config.configurations.default.

`2.7.4 :007 > locale = ''
 => "" 
2.7.4 :008 > locale ||= 'abc'
 => "" 
2.7.4 :009 > locale
 => "" 
`